server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -836,4 +852,11 @@ private void homerMovesTheMailInAnotherMailbox(String messageId) {
             .when()
             .post("/jmap");
     }
+
+    private boolean homerHasMailboxWithRole(Role role) {
+        return getAllMailboxesIds(homerAccessToken).stream()
+            .filter(mailbox -> mailbox.get("role") != null)
+            .anyMatch(mailbox -> mailbox.get("role").equalsIgnoreCase(role.serialize())
```

Comment:
From RoleTest I see you expect Role are case sensitive, then I think this role comparing here should be strictly equals instead of equalsIgnoreCase